smoke-tests/apps/SamplingOverrides/src/smokeTest/resources/applicationinsights-thread-id-and-name.json

```diff
@@ -0,0 +1,86 @@
+{
+  "connectionString": "InstrumentationKey=12345678-0000-0000-0000-0FEEDDADBEEF;IngestionEndpoint=http://host.testcontainers.internal:6060/",
+  "role": {
+    "name": "testrolename",
+    "instance": "testroleinstance"
+  },
+  "sampling": {
+    "percentage": 50
+  },
+  "preview": {
+    "sampling": {
+      "overrides": [
+        {
+          "telemetryType": "request",
+          "attributes": [
+            {
+              "key": "thread.name",
+              "value": "http-nio-.*",
+              "matchType": "regexp"
+            },
+            {
+              "key": "http.target",
+              "value": "/SamplingOverrides/health-check",
+              "matchType": "strict"
+            }
+          ],
+          "percentage": 100
+        },
+        {
+          "telemetryType": "request",
+          "attributes": [
+            {
+              "key": "thread.name",
+              "value": "http-nio-.*",
+              "matchType": "regexp"
+            },
+            {
+              "key": "http.target",
+              "value": "/SamplingOverrides/",
+              "matchType": "strict"
+            }
+          ],
+          "percentage": 100
+        },
+        {
+          "telemetryType": "request",
+          "attributes": [
+            {
+              "key": "thread.name",
+              "value": "http-nio-.*",
+              "matchType": "regexp"
+            },
+            {
+              "key": "http.target",
+              "value": "/SamplingOverrides/thread-id-and-name",
+              "matchType": "strict"
+            }
+          ],
+          "percentage": 0
+        },
```

Comment:
I think this one can be removed in favor of the one below

Reply:
the one below is trace

Reply:
oh, i missed that, thx!